### PR TITLE
Add WNP 130 (Fixes #14946)

### LINKED ISSUE #14946: WNP 130 EU + NA – Duplicate Tabs
Hello. Please refer to **[this](https://docs.google.com/document/d/1uYDZGFlbSHBiI_vZn8DhvgCEo9sL2Y3_yljnsfPdBEs/edit)** document for our brief on WNP 130 EU + NA (internal only). 

**Go-live Date:** goes live on Sept 3, 2024
**Web receives final assets:** August 21, 2024

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP 130

## Issue / Bugzilla link

#14946)

## Testing

- http://localhost:8000/en-US/firefox/130.0/whatsnew/ (EN-US)
- http://localhost:8000/en-US/firefox/130.0/whatsnew/ (EN-CA)
- http://localhost:8000/en-US/firefox/130.0/whatsnew/ (EN-GB)
- http://localhost:8000/de/firefox/130.0/whatsnew/ (DE)
- http://localhost:8000/fr/firefox/130.0/whatsnew/ (FR)
- http://localhost:8000/es-ES/firefox/130.0/whatsnew/ (ES)
- http://localhost:8000/it/firefox/130.0/whatsnew/ (IT)
- http://localhost:8000/pl/firefox/130.0/whatsnew/ (PL)

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx130.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          'l10n': True
+      }
+    ) }}
+
+    <h1 class="wnp-main-title">{{ main_title }}</h1>
+
+    <p class="wnp-main-tagline">{{ main_tagline }}</p>
+
+    <p class="wnp-main-cta">
+      <a class="mzp-c-button mzp-t-product" href="#TODO" data-cta-type="button" data-cta-text="Here’s how">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx130.html:89]: Note to self: CTA still needs link

File: media/img/l10n/en-US/firefox/wnp-130/duplicate-tabs-1000.gif
Comment (alexgibson) [inline media/img/l10n/en-US/firefox/wnp-130/duplicate-tabs-1000.gif:1]: Note to self: need to add L10n images when they're ready

**Comment by alexgibson** [comment]
There's a localized image still to come, but otherwise this is ready for code review.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx130.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          'l10n': True
+      }
+    ) }}
+
+    <h1 class="wnp-main-title">{{ main_title }}</h1>
+
+    <p class="wnp-main-tagline">{{ main_tagline }}</p>
+
+    <p class="wnp-main-cta">
+      <a class="mzp-c-button mzp-t-product" href="{{ cta_link }}" data-cta-type="button" data-cta-text="Here’s how">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx130.html:95]: **Nit:** Please remove `data-cta-type`

**Review by stephaniehobson — APPROVED** [review]
R+ 👯

**Pull request merged**